src/components/popover-container/popover-container-content.style.js

```diff
@@ -0,0 +1,21 @@
+import styled, { css } from 'styled-components';
+import { baseTheme } from '../../style/themes';
+
+const PopoverContainerContentStyle = styled.div`
+  ${({ theme }) => theme && css`
+    background: ${theme.colors.white};
```

Comment:
Is there a need for the `css` util? It's not really complicated so you can just do two interpolations